cleverhans/attacks/projected_gradient_descent.py

```diff
@@ -175,7 +177,7 @@ def parse_params(self,
                    clip_max=None,
                    y_target=None,
                    rand_init=None,
-                   rand_minmax=0.3,
```

Comment:
This argument seems to have been added specifically for the l-inf case. We could make this a more general argument that specifies the size (in either linf or l2 norm) of the initial random start. But does anyone really use this? I don't see a reason why you wouldn't start from a random point within the full linf or l2 ball. Unless this can be used to implement R+FGSM?

Reply:
Yeah I remember a paper using a ball of a smaller radius to do the random start. I'd rather keep it if you don't need to remove it and it's not too much work; since this version of the attack will be used to benchmark new implementations in tf2 / pytorch / jax as we move forward to implementations in new frameworks.

Reply:
I've added this parameter back, renamed as rand_init_eps which should be more evocative of the general case. By default, this parameter takes the same value as eps.